### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by timosachsenberg**
some minor extensions

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+There are many ways how you can contribute:
   - write/update documentation
-  - fix bugs
-  - ...
-  
-To incorporate your contributions, a pull request against the OpenMS 'develop' branch is probably
-the most efficient way. A core developer will review your changes and approve them (or ask for modifications).
+  - implement new algorithms, features or tools
+  - fix bugs and resolve issues
+  - many more...
```
Comment (hannesveit) [inline CONTRIBUTING.md]: like what? :)

Comment (hroest) [inline CONTRIBUTING.md]: As this is the message that people see when creating the first PR we might want to be more specific in what they should do and not do

**Comment by timosachsenberg** [comment]
I think we need to first decide on the aims of the file.
Is it primarily intended for people already opening a PR - then some of the current information would not make sense. Or is it also a general document to on how you can contribute to OpenMS.
I would assume that it is primarily for people opening a PR and then the document should focus on what the contributor has to expect from the review process. Also it should point to coding convention, how to write a commit message, how to add tests, ... to provide reasons why he/she might need to perform additional work.

**Comment by timosachsenberg** [comment]
some examples are linked from: https://github.com/blog/1184-contributing-guidelines

**Comment by timosachsenberg** [comment]
ok I added more stuff. Please feel free to extend and correct it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,13 +1,43 @@
...
+  - the platform you are running OpenMS on (Windows 10, ...)
+  - how you installed OpenMS (e.g., from within KNIME, binary installers, self compiled)
+  - a description on how to reproduce the bug
+  - relevant tool output (e.g., error messages)
+  - data to repoduce the bug (If possible as a github gist. Other platforms like dropbox, google drive links also work. If you can't share the data publicly please indicate this and we will contact you in private.)
+
+Optionally:
+  - label your issue using github labels (e.g. as: question, defect) that indicated the type of issue and which components of OpenMS (blue labels) are affected. The severity is usually assigned by OpenMS maintainers and used internally to e.g. indicate if a bug is a blocker for a new release.
+
+#Opening a Pull Request
```
Comment (cbielow) [inline CONTRIBUTING.md]: user might not know what whis is so
`#Opening a Pull Request (for programmers/developers)`
?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
+  - request new features
   - write/update documentation
-  - fix bugs
-  - ...
-  
-To incorporate your contributions, a pull request against the OpenMS 'develop' branch is probably
-the most efficient way. A core developer will review your changes and approve them (or ask for modifications).
 
-Further details on our GitHub-Wiki: https://github.com/OpenMS/OpenMS/wiki#-for-developers
+If you are a programmer you can open a Pull Request to:
```
Comment (cbielow) [inline CONTRIBUTING.md]: maybe move this below `# Opening a PR`?

**Comment by timosachsenberg** [comment]
ok I squash and merge this as a single commit. Please feel free to refine the document.

**Pull request merged**